packages/core/phase1/enrichAho/enrichFunctions/enrichWithPncQuery.ts

```diff
@@ -70,33 +69,37 @@ export default async (
     "PNC Response Time": new Date().getTime() - requestStartTime.getTime(),
     "PNC Attempts Made": 1, // Retry is not implemented
     "PNC Request Type": "enquiry",
-    "PNC Request Message":
-      annotatedHearingOutcome.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.ArrestSummonsNumber,
+    "PNC Request Message": aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.ArrestSummonsNumber,
     "PNC Response Message": isError(pncResult) ? pncResult.messages.join(", ") : pncResult,
     sensitiveAttributes: "PNC Request Message,PNC Response Message"
   }
 
   auditLogger.info(EventCode.PncResponseReceived, auditLogAttributes)
   if (isError(pncResult)) {
     for (const message of pncResult.messages) {
-      if (!isIgnored && (!isNotFoundError(message) || isCaseRecordable(annotatedHearingOutcome))) {
-        annotatedHearingOutcome.Exceptions.push(generatePncEnquiryExceptionFromMessage(message))
+      if (!isIgnored && (!isNotFoundError(message) || isCaseRecordable(aho))) {
+        aho.Exceptions.push(generatePncEnquiryExceptionFromMessage(message))
       }
     }
   } else {
-    annotatedHearingOutcome.PncQuery = pncResult
+    aho.PncQuery = pncResult
   }
 
-  annotatedHearingOutcome.PncQueryDate = pncGateway.queryTime
+  aho.PncQueryDate = pncGateway.queryTime
 
-  addTitleToCaseOffences(annotatedHearingOutcome.PncQuery?.courtCases)
-  addTitleToCaseOffences(annotatedHearingOutcome.PncQuery?.penaltyCases)
+  addTitleToCaseOffences(aho.PncQuery?.courtCases)
+  addTitleToCaseOffences(aho.PncQuery?.penaltyCases)
 
-  if (annotatedHearingOutcome.PncQuery !== undefined) {
-    annotatedHearingOutcome.AnnotatedHearingOutcome.HearingOutcome.Case.RecordableOnPNCindicator = true
+  if (aho.PncQuery !== undefined) {
+    aho.AnnotatedHearingOutcome.HearingOutcome.Case.RecordableOnPNCindicator = true
   }
 
-  annotatedHearingOutcome = matchOffencesToPnc(annotatedHearingOutcome)
+  aho = matchOffencesToPnc(aho)
 
-  return annotatedHearingOutcome
+  const { PNCIdentifier, CourtPNCIdentifier } = aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant
```

Comment:
How sure are we there are no side effects from doing this? I don't think old Bichard did this as the comparison tests were passing so we need to be sure of the impact

Reply:
Or is this just moved from `matchOffencesToPnc`?

Reply:
Ben and I discussed this on a call - the old system was designed not to set these fields if there were exceptions, which is something we copied into core. This change causes these fields to be set all the time, which is something that's been queried in feedback from users.

We don't believe it will have any negative impact.